20080602, Stuart Hughes <stuarth at freescale dot com>

Enabled local user ftp accounts and enable write and set a more useful
umask

--- vsftpd-2.0.5/vsftpd.conf	2005-05-20 23:52:54.000000000 +0100
+++ vsftpd-2.0.5.modified/vsftpd.conf	2008-05-27 15:14:18.000000000 +0100
@@ -12,14 +12,14 @@
 anonymous_enable=YES
 #
 # Uncomment this to allow local users to log in.
-#local_enable=YES
+local_enable=YES
 #
 # Uncomment this to enable any form of FTP write command.
-#write_enable=YES
+write_enable=YES
 #
 # Default umask for local users is 077. You may wish to change this to 022,
 # if your users expect that (022 is used by most other ftpd's)
-#local_umask=022
+local_umask=022
 #
 # Uncomment this to allow the anonymous FTP user to upload files. This only
 # has an effect if the above global write enable is activated. Also, you will
